packages/bikram-sambat/src/index.ts

```diff
@@ -363,4 +366,46 @@ export default class BikramSambat implements BikramSambatProps {
 
     return dayjs(this.adDate).isAfter(date, unit);
   }
+  /**
+   * Checks if the current BikramSambat date is after the given date.
```

Comment:
```suggestion
   * Checks if the current BikramSambat date is between the given dates. 
```
Add some infor about boundary inclusion